docs/_documentation/using_authorization.md

```diff
@@ -74,10 +85,24 @@ curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/event-t
 }'
 ```
 
+For creating a subscription, the sample request is as follows:
+
+```bash
+curl -v -XPOST -H "Content-Type: application/json" http://localhost:8080/subscriptions/ea2d7472-ddc6-4b9e-91f1-5bcd0b7b4fa4 -d '{
```

Comment:
Why is the subscription id in the URL, if this is the creation of a subscription?

Reply:
Oops, messed up. Updated in `7265d6e`.